java/server/src/org/openqa/selenium/grid/graphql/selenium-grid-schema.graphqls

```diff
@@ -14,6 +14,17 @@ type Session {
   id: String!
   capabilities: String!
   startTime: String!
+  uri: Uri!
+  nodeId: String!
+  nodeUri: Uri!
+  sessionDurationMillis: String!
+  slot: Slot
```

Comment:
This seems back-to-front --- the slot owns the session. I guess it's alright for now, but it's a little odd.